report_thesis/src/sections/background.tex

```diff
@@ -148,6 +150,103 @@ \subsubsection{Partial Least Squares (PLS)}
 The components are then used to predict the target variable by fitting a linear model via least squares regression.
 
 \subsubsection{Support Vector Regression (SVR)}
+\gls{svr} is a regression technique that extends the principles of \gls{svm} to regression problems.
+We therefore provide an overview of \gls{svm}s based on \citet{James2023AnIS} before discussing \gls{svr}s.
+
+\gls{svm} is a supervised learning algorithm used primarily for classification tasks.
+A core concept in \gls{svm} is the \textit{hyperplane}.
+Generally, a hyperplane is a subspace of one dimension less than its ambient space.
+This means that in a two-dimensional space, a hyperplane is a line, while in a three-dimensional space, it is a plane, and so on.
+
+\gls{svm} is built on the idea of finding the hyperplane that best separates the data points into different classes.
+This hyperplane is chosen to maximize the margin, which is the distance between the hyperplane and the nearest data point from either class.
+The data points that lie closest to the hyperplane are called \textit{support vectors}.
+
+% Mathematically, \gls{svm} solves the optimization problem:
```

Comment:
There are multiple ways to train SVM classifiers, and explaining the math would take the entire duration of our exam. For example, you provide only the definition here for soft-margin SVMs (apparent at a glance from the $\xi$ slack variable). There's also hard-margin SVM. And there's the entirely different way of solving it, by solving the dual problem. And that isn't even mentioning kernelized SVMs.

I vote to keep it out of the report so we can actually focus on the contents of our paper during the exam this time...

Reply:
Agreed.